Doc/whatsnew/3.14.rst

```diff
@@ -1094,135 +1039,173 @@ Platform support
-  :exc:`SyntaxError`. (Contributed by Irit Katriel in :gh:`122245`.)
+* More syntax errors are now detected regardless of optimisation and
+  the :option:`-O` command-line option.
+  This includes writes to ``__debug__``, incorrect use of :keyword:`await`,
+  and asynchronous comprehensions outside asynchronous functions.
+  For example, ``python -O -c 'assert (__debug__ := 1)'``
+  or ``python -O -c 'assert await 1'`` now produce :exc:`SyntaxError`\ s.
+  (Contributed by Irit Katriel and Jelle Zijlstra in :gh:`122245` & :gh:`121637`.)
 
-* Add class methods :meth:`float.from_number` and :meth:`complex.from_number`
-  to convert a number to :class:`float` or :class:`complex` type correspondingly.
-  They raise an error if the argument is a string.
-  (Contributed by Serhiy Storchaka in :gh:`84978`.)
+* When subclassing a pure C type, the C slots for the new type
+  are no longer replaced with a wrapped version on class creation
+  if they are not explicitly overridden in the subclass.
+  (Contributed by Tomasz Pytel in :gh:`132284`.)
 
-* Implement mixed-mode arithmetic rules combining real and complex numbers as
-  specified by C standards since C99.
-  (Contributed by Sergey B Kirpichev in :gh:`69639`.)
 
-* All Windows code pages are now supported as "cpXXX" codecs on Windows.
-  (Contributed by Serhiy Storchaka in :gh:`123803`.)
+Built-ins
+---------
 
-* :class:`super` objects are now :mod:`pickleable <pickle>` and
-  :mod:`copyable <copy>`.
-  (Contributed by Serhiy Storchaka in :gh:`125767`.)
+* The :func:`bytes.fromhex` and :func:`bytearray.fromhex` methods now accept
+  ASCII :class:`bytes` and :term:`bytes-like objects <bytes-like object>`.
+  (Contributed by Daniel Pope in :gh:`129349`.)
 
-* The :class:`memoryview` type now supports subscription,
-  making it a :term:`generic type`.
-  (Contributed by Brian Schubert in :gh:`126012`.)
+* Add class methods :meth:`float.from_number` and :meth:`complex.from_number`
+  to convert a number to :class:`float` or :class:`complex` type correspondingly.
+  They raise a :exc:`TypeError` if the argument is not a real number.
+  (Contributed by Serhiy Storchaka in :gh:`84978`.)
 
 * Support underscore and comma as thousands separators in the fractional part
   for floating-point presentation types of the new-style string formatting
   (with :func:`format` or :ref:`f-strings`).
   (Contributed by Sergey B Kirpichev in :gh:`87790`.)
 
-* The :func:`bytes.fromhex` and :func:`bytearray.fromhex` methods now accept
-  ASCII :class:`bytes` and :term:`bytes-like objects <bytes-like object>`.
-  (Contributed by Daniel Pope in :gh:`129349`.)
-
-* Support ``\z`` as a synonym for ``\Z`` in :mod:`regular expressions <re>`.
-  It is interpreted unambiguously in many other regular expression engines,
-  unlike ``\Z``, which has subtly different behavior.
-  (Contributed by Serhiy Storchaka in :gh:`133306`.)
+* The :func:`int` function no longer delegates to :meth:`~object.__trunc__`.
```

Comment:
`int` obviously isn't actually a function, but here I think saying 'function' reads better than 'The `int` built-in ...'.